tests/library_closure_error.js

```diff
@@ -0,0 +1,6 @@
+mergeInto(LibraryManager.library, {
+	foo: function() {
+		thisVarDoesNotExist++;
+		thisVarDoesNotExistEither++;
+	}
```

Comment:
Could be simpler to keep this all in a single file and just use `EM_ASM`?    Not a big deal I guess.